langchain4j-oracle/src/main/java/dev/langchain4j/store/embedding/oracle/OracleEmbeddingStore.java

```diff
@@ -11,17 +11,14 @@
 import dev.langchain4j.store.embedding.EmbeddingStore;
 import dev.langchain4j.store.embedding.filter.Filter;
 import lombok.Builder;
+import oracle.jdbc.OracleStatement;
 import oracle.jdbc.OracleType;
+import oracle.jdbc.OracleTypes;
 import oracle.sql.json.OracleJsonObject;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.math.BigDecimal;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
```

Comment:
I know this is IntelliJ probably.

Reply:
We should have a general discussion about the code style for this module

Normally, we don't use * imports. Nor do we use 120 character lines, nor do we use 4 spaces of indentation. Personally, I would prefer to follow our usual conventions rather than these. But I didn't overhaul the style because of:
>Follow existing code style present in the project.

Found in [CONTRIBUTING.MD](https://github.com/psilberk/langchain4j/blob/main/CONTRIBUTING.md)

Although I don't like it, I think we need to follow the style in order to be good contributors.

To configure IntelliJ, I went to Settings -> Editor -> Code Style. There is a "Scheme" option at the top. I set this to "Project". Seems that IntelliJ picked up the existing style somehow. Not sure if there's a configuration file somewhere, or if Idea just figures it out automatically.

Reply:
Agreed. They don't seem to have a code style scheme or anything like it, right? (I couldn't find it). Otherwise we just need to follow their style based on the existing code?

Reply:
Yes, it's basically a best effort situation: Look at the existing code and do your best to match the style.
There is no detailed style guideline.